invenio/modules/formatter/format_elements/bfe_edit_record.py

```diff
@@ -39,12 +41,14 @@ def format_element(bfo, style):
         linkattrd = {}
         if style != '':
             linkattrd['style'] = style
+        if html_class != '':
+            linkattrd['class'] = html_class
         out += create_html_link(
             CFG_BASE_URL +
             '/%s/edit/?ln=%s#state=edit&recid=%s' % (CFG_SITE_RECORD, bfo.lang,
                                                      str(bfo.recID)),
             {},
-            link_label=_("Edit This Record"),
```

Comment:
if you remove it from `_(...)` then it will not be extracted for translations.

```python
from invenio.base.i18n import _

def ..., html_class=None, link_label=_(...)...
    ...
    if html_class:
    ...
```